server/constants/messages.go

```diff
@@ -25,4 +25,7 @@ const (
 	AlreadyLinkedProject            = "This project is already linked."
 	GetProjectListError             = "Error getting Project List"
 	ErrorFetchProjectList           = "Error in fetching project list"
+	ErrorDecodingBody               = "Error in decoding body"
+	ProjectNotFound                 = "Requested project does not exists"
```

Comment:
```suggestion
	ProjectNotFound                 = "Requested project does not exist"
```

Reply:
`exists` is a singular here as we are checking for 1 project only at a time

Reply:
I know and the correct grammar is `exist` here because we are using it with `does not`.